config/import-control.xml

```diff
@@ -59,9 +59,12 @@
     <allow class="java.math.BigInteger"/>
   </file>
 
+  <file name="ParserUtil">
+    <allow class="antlr.Token"/>
+    <allow class="com.puppycrawl.tools.checkstyle.DetailAstImpl"/>
+  </file>
+
   <subpackage name="utils">
-    <!-- until https://github.com/checkstyle/checkstyle/issues/9272 -->
-    <allow pkg="antlr"/>
     <!-- until https://github.com/checkstyle/checkstyle/issues/9271 -->
     <allow class="com.puppycrawl.tools.checkstyle.DetailAstImpl"/>
```

Comment:
this line should be removed as well

Reply:
Shouldn't this be removed in #9271 ?

Reply:
no, it was messed up a little. You can remove this line and reference to issue as well

Reply:
Issue is about different line in this file.
@nmancus1 , please try to remove this allowance in until package.

Reply:
Done.